VideoLocker/res/layout/discussion_responses_thread_row.xml

```diff
@@ -1,91 +1,82 @@
-                android:layout_alignParentStart="true"
-                android:layout_alignParentLeft="true"
-                android:layout_centerVertical="true"
                 android:layout_marginEnd="@dimen/discussion_responses_thread_text_view_spacing"
                 android:layout_marginRight="@dimen/discussion_responses_thread_text_view_spacing"
+                android:visibility="gone"
                 app:iconColor="@color/edx_grayscale_neutral_base"
-                app:iconName="fa_thumb_tack"
-                android:visibility="gone"/>
-
-            <RelativeLayout
-                android:id="@+id/discussion_author_relative_layout"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_toEndOf="@+id/discussion_responses_thread_row_pinned_icon_view"
-                android:layout_toRightOf="@+id/discussion_responses_thread_row_pinned_icon_view">
+                app:iconName="fa_lock" />
 
-                <include layout="@layout/discussion_author_layout" />
-
-            </RelativeLayout>
+            <org.edx.mobile.third_party.iconify.IconView
+                android:id="@+id/discussion_responses_thread_row_pinned_icon_view"
+                android:layout_width="@dimen/edx_xxx_small"
+                android:layout_height="@dimen/edx_xxx_small"
+                android:layout_marginEnd="@dimen/discussion_responses_thread_text_view_spacing"
+                android:layout_marginRight="@dimen/discussion_responses_thread_text_view_spacing"
+                android:visibility="gone"
+                app:iconColor="@color/edx_grayscale_neutral_base"
+                app:iconName="fa_thumb_tack" />
 
-            <RelativeLayout
-                android:id="@+id/discussion_number_responses_relative_layout"
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:layout_alignParentEnd="true"
-                android:layout_alignParentRight="true"
-                android:gravity="end"
-                android:layout_toRightOf="@+id/discussion_author_relative_layout">
+            <include layout="@layout/discussion_author_layout" />
 
-                <include layout="@layout/number_responses_or_comments_layout" />
+            <View
+                android:layout_width="0dp"
+                android:layout_height="0dp"
+                android:layout_weight="1" />
 
-            </RelativeLayout>
+            <include layout="@layout/number_responses_or_comments_layout" />
 
-        </RelativeLayout>
+        </LinearLayout>
 
-    </RelativeLayout>
+    </LinearLayout>
 
     <View
-        android:id="@+id/discussion_responses_divider"
-        android:layout_width="match_parent"
-        android:layout_height="1dp"
+        style="@style/separator_base"
         android:background="@color/edx_grayscale_neutral_xx_light" />
```

Comment:
The design doc doesnt differentiate any dividers and only includes the color `edx_grayscale_neutral_light` for it.
Here's the mockup for the screen: https://projects.invisionapp.com/share/P22Y3C25W#/screens/77281493, which shows the color to be `#F2F2F2`